### ISSUE TITLE
onTap doesn't work

### ISSUE THREAD

**Issue by guilhermeeng99** [peiffer-innovations/websafe_svg#3]
when WebsafeSvg.asset is a child of any onTap, onTap does not work. 

Example: 

```
InkWell(
          onTap: () => print('test'),
          child: Container(
            height: 28,
            width: 28,
            child: WebsafeSvg.asset(
              svgPath,
            ),
          ),
),
```

**Comment by jpeiffer** [comment]
There's nothing that should prevent a parent widget from working this way.  I'm adding an example tonight and will include an example of `onTap` in it to confirm.

**Comment by jpeiffer** [comment]
This is fixed and working with version `1.10.0`.  The new example app provides a working example as well.

If you find there is still a problem, feel free to reopen this issue, or simply open a new one with the new details.
